### DISCUSSION TITLE
如何获得xmake的lua标准库提示????

### DISCUSSION THREAD

**Discussion by JuanZoran** [xmake-io/xmake#3571] [Q&A]
类似于获取neovim的标准库提示, 使用neodev.nvim可以正确的设置好nvim的标准库函数信息, 参数解释的信息, 这大大的帮助了我在编写简单nvim lua脚本,对于xmake.lua, 我想知道有没有类似的插件或者lua-language-server的设置, 可以lsp读取到xmake的标准库, 
![image](https://user-images.githubusercontent.com/107862700/228137493-2a9ddab4-34e2-49b9-a8a4-c669099c852d.png)
![image](https://user-images.githubusercontent.com/107862700/228137574-988cb169-c416-49cc-8654-54562a3e8ad6.png)

我尝试了将$HOME/.local/share/xmake设置为为标准库路径, 但是还是无法正确解析target函数的信息, 似乎target这些全局函数并不是在这里定义的, 或者是有其他的方式

**Comment by waruqi** [comment]
相关的一些东西可以参考下

https://github.com/xmake-io/xmake-vscode/issues/172
https://github.com/LelouchHe/xmake-luals-addon
https://github.com/SpaceVim/SpaceVim/pull/4808

**Comment by JuanZoran** [comment]
感谢
